docs/DeveloperGuide.md

```diff
@@ -278,8 +278,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | Teaching Assistant                                        | add students to TAPA                                                                   | know which students are taking my classes                                                     |
 | `* * *`  | Teaching Assistant                                        | delete a specific student from TAPA                                                    | remove student who has withdrawn from the module during the semester                          |
 | `* * *`  | Teaching Assistant                                        | list students alphabetically                                                           | print students in a list as their names would appear on the official class list               |
-| `* * *`  | Teaching Assistant                                        | save existing data to hard drive                                                       | access the data again when I reopen the app                                                   |
+| `* * *`  | Teaching Assistant                                        | tag students                                                                           | know what modules they are taking                                                             |
+| `* * *`  | Teaching Assistant                                        | filter students by their tags                                                          | view the progress of students in each separate tag (different class/module)                   |
+| `* * *`  | Teaching Assistant                                        | define my own tasks for each module                                                    | customise the tasks to each module                                                            |
+| `* * *`  | Teaching Assistant                                        | assign tasks to multiple students                                                      | allocate tasks to students without having to individually assign them                         |
+| `* * *`  | Teaching Assistant                                        | undo the most recent command entered                                                   | revert any unwanted changes (such as deleting student) without much trouble                   |
+| `* * *`  | Teaching Assistant                                        | delete a task given to a student                                                       | delete the task from the list of tasks                                                        |
```

Comment:
```suggestion
| `* * *`  | Teaching Assistant                                        | delete a task given to a student                                                       | remove task that has incorrect information or has been assigned wrongly                                                        |
```
Feel a little weird to repeat the word "delete" in both part 2 and 3.

Remember to update the docs as well if you decide to change

Reply:
Edited it to make it sound more natural.